pkgs/tools/networking/mcrcon/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub
+, gcc }:
+
+stdenv.mkDerivation rec {
+  name = "mcrcon-${version}";
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "Tiiffi";
+    repo = "mcrcon";
+    rev = "v${version}";
+    sha256 = "1pwr1cjldjy8bxqpp7w03nvdpw8l4vqfnk6w6b3mf0qpap1k700z";
+  };
+
+  buildInputs = [ gcc ];
```

Comment:
are you sure there is a hard dependency on gcc? If not you can just remove this line and use the command as shown above.